src/FactoryTrait.php

```diff
@@ -142,7 +142,7 @@ public function factory($seed, $defaults = [], string $prefix = null): object
         $object = array_shift($arguments); // first numeric key argument is object
         if (!$object) {
             throw new Exception([
-                'factory() could not find object',
+                'factory() could not find object in seed',
```

Comment:
@DarkSide666 

![image](https://user-images.githubusercontent.com/2228672/80498653-48dae600-896c-11ea-8a84-375614a98c18.png)

Reply:
ah i see. ok. if object===null, then it will throw exception below anyway.